forex-service/src/main/java/lk/ac/ucsc/forexservice/ForexController.java

```diff
@@ -0,0 +1,28 @@
+package lk.ac.ucsc.forexservice;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ForexController {
+    @Autowired
+    private Environment environment;
```

Comment:
@Binurim field injectors should be avoided if possible

Reply:
Will look into this. Thank you!